cobigen-eclipse/cobigen-eclipse-test/src/main/java/com/devonfw/cobigen/eclipse/test/AdaptTemplatesTest.java

```diff
@@ -80,31 +80,24 @@ public void testBasicOpenAPIGenerationWithAdaptTemplates() throws Exception {
     SWTBotTreeItem javaClassItem = view.bot().tree().expandNode(testProjName, "adapt-templates.yml");
     javaClassItem.select();
 
-    // execute CobiGen
-    EclipseCobiGenUtils.processCobiGen(bot, javaClassItem, 25000, "CRUD devon4j Server>CRUD REST services");
-    bot.waitUntil(new AllJobsAreFinished(), 10000);
-    // increase timeout as the openAPI parser is slow on initialization
-    EclipseCobiGenUtils.confirmSuccessfullGeneration(bot, 40000);
-
     bot.waitUntil(new AllJobsAreFinished(), 10000);
-    IProject proj = ResourcesPlugin.getWorkspace().getRoot().getProject(testProjName);
-    IFile generationResult = proj.getFile(
-        "src/main/java/com/devonfw/test/sampledatamanagement/service/impl/rest/SampledatamanagementRestServiceImpl.java");
+    String Cobigen_templates = ConfigurationConstants.COBIGEN_TEMPLATES;
+    IProject adaptedTemplatesProj = ResourcesPlugin.getWorkspace().getRoot().getProject(Cobigen_templates);
```

Comment:
Why not directly use the constant `ConfigurationConstants.COBIGEN_TEMPLATES` here